packages/bsky/src/hydration/feed.ts

```diff
@@ -86,7 +89,11 @@ export class FeedHydrator {
     return need.reduce((acc, uri, i) => {
       const record = parseRecord<PostRecord>(res.records[i], includeTakedowns)
       const violatesThreadGate = res.meta[i].violatesThreadGate
-      return acc.set(uri, record ? { ...record, violatesThreadGate } : null)
+      const violatesQuotegate = res.meta[i].violatesQuoteGate
```

Comment:
Coming from the dataplane this will be violatesQuoteGate, I think bc the buf gen converts snake to camel. We could probably also rename the proto to `violates_quotegate`, but figured might as well keep it consistent in the dataplane.

Everywhere else I refer to it as violatesQuotegate.